### PR TITLE
Add demo script and update the README

### PR DESCRIPTION
**Pull request by dkorn**

### CODE REVIEW

File: demo.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# GET Fails to authenticate with bad User API Token
```
Comment (erikzaadi) [inline demo.sh]: Y U NO HASHBANGZ

File: demo.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+http localhost:1449/api/incidents Authorization:blah
+
+# GET all incidents with administrator User API token (Admin role)
```
Comment (erikzaadi) [inline demo.sh]: Mayhaps echo this before doing the request, with a little `read -p` magic?

**Review by erikzaadi — APPROVED** [review]
Small comments, but looks awesomeico

File: demo.sh (hunk 1 above)
Comment (dkorn) [reply demo.sh]: *embarrassed*
added, thanks!

File: demo.sh (hunk 2 above)
Comment (dkorn) [reply demo.sh]: great suggestion, did a bit more ;)

**Pull request closed without merging**